public/css/customv600.css

```diff
@@ -18,15 +25,20 @@
     color:#555555;
 }
 
-.navMove {
-    display:flex;
-    justify-content:center;
-    align-items:center;
+.navItem {
```

Comment:
I separated out the `navMove` into two.

`navItem` is any item in the navigator component, including the move number.
`navMove` is for white and blacks moves only, and contains the hover styling.